segmentation/utils/spatial_analysis.py

```diff
@@ -53,202 +52,203 @@ def calculate_channel_spatial_enrichment(dist_matrices, marker_thresholds, all_d
         raise ValueError("Threshold Markers do not match markers in Expression Matrix")
 
     # Create stats Xarray with the dimensions (points, stats variables, number of markers, number of markers)
     stats_raw_data = np.zeros((num_fovs, 7, marker_num, marker_num))
-    coords = [fovs, ["z", "muhat", "sigmahat", "p_pos", "p_neg", "h", "p_adj"], marker_titles,
+    coords = [included_fovs, ["z", "muhat", "sigmahat", "p_pos", "p_neg", "h", "p_adj"], marker_titles,
               marker_titles]
     dims = ["points", "stats", "marker1", "marker2"]
     stats = xr.DataArray(stats_raw_data, coords=coords, dims=dims)
 
     # Subsetting threshold matrix to only include column with threshold values
     thresh_vec = marker_thresholds.iloc[:, 1]
 
-    for i in range(0, len(fovs)):
-        # Subsetting expression matrix to only include patients with correct label
-        patient_ids = all_data[fov_col] == fovs[i]
-        fov_data = all_data[patient_ids]
-        # Patients with correct label, and only columns of markers
-        fov_channel_data = data_markers[patient_ids]
+    for i in range(0, len(included_fovs)):
+        # Subsetting expression matrix to only include patients with correct fov label
+        current_fov_idx = all_fov_data[fov_col] == included_fovs[i]
+        current_fov_data = all_fov_data[current_fov_idx]
+        # Patients with correct label, and only columns of channel markers
+        current_fov_channel_data = all_channel_data[current_fov_idx]
 
         # Subset the distance matrix dictionary to only include the distance matrix for the correct point
-        dist_matrix = dist_matrices[str(fovs[i])]
+        dist_matrix = dist_matrices_dict[str(included_fovs[i])]
 
         # Get close_num and close_num_rand
         close_num, marker1_num, marker2_num = spatial_analysis_utils.compute_close_cell_num(
             dist_mat=dist_matrix, dist_lim=100, num=marker_num, analysis_type="Channel",
-            fov_data=fov_data, fov_channel_data=fov_channel_data, thresh_vec=thresh_vec)
+            current_fov_data=current_fov_data, current_fov_channel_data=current_fov_channel_data,
+            thresh_vec=thresh_vec)
 
         close_num_rand = spatial_analysis_utils.compute_close_cell_num_random(
             marker1_num, marker2_num, dist_matrix, marker_num, dist_lim, bootstrap_num)
         values.append((close_num, close_num_rand))
 
         # Get z, p, adj_p, muhat, sigmahat, and h
         stats_xr = spatial_analysis_utils.calculate_enrichment_stats(close_num, close_num_rand)
-        stats.loc[fovs[i], :, :] = stats_xr.values
+        stats.loc[included_fovs[i], :, :] = stats_xr.values
     return values, stats
 
 
-def calculate_cluster_spatial_enrichment(all_data, dist_mats, fovs=None,
-                                         bootstrap_num=1000, dist_lim=100):
+def calculate_cluster_spatial_enrichment(all_fov_data, dist_matrices_dict, included_fovs=None,
```

Comment:
Same thing here, these are all great changes, just swap all_fov_data back to all_data